src/styles/styles.css

```diff
@@ -253,327 +253,3 @@ body .hline {
 .form-control:-ms-input-placeholder {
   color: lightgray;
 }
-
```

Comment:
moving component-wise styles to their respective component css, keep only the global css in styles.css